src/System.Net.NetworkInformation/src/Configurations.props

```diff
@@ -5,7 +5,7 @@
       uap-Windows_NT;
       netcoreapp-Linux;
       netcoreapp-OSX;
-      netcoreapp-Unix;
+      netcoreapp-FreeBSD;
```

Comment:
I wonder if the netcoreapp-Unix should have stayed here as well. The configurations in the .csproj below refer to it.